### PR TITLE
[vULR7sSU] Update APOC descriptions

### PR DESCRIPTION
**Pull request by gem-neo4j**
Update missed and malformed descriptions

### CODE REVIEW

File: core/src/main/java/apoc/nodes/Nodes.java [hunk 1]
```diff
@@ -653,20 +653,20 @@ public boolean isDense(@Name("node") Node node) {
     }
 
     @UserFunction("apoc.any.isDeleted")
-    @Description("returns boolean value for nodes and rele existance")
-    public boolean isDeleted(@Name("thing") Object thing) {
-        if (thing == null) return true;
+    @Description("Returns true if the given node or relationship no longer exists.")
```
Comment (gem-neo4j) [inline core/src/main/java/apoc/nodes/Nodes.java:656]: This is just my copy for it, I'll get Jens to double check it (the original is so bad I couldn't leave it 🙈)

File: core/src/main/java/apoc/coll/Coll.java [hunk 2]
```diff
@@ -963,13 +964,10 @@ public List<List<Object>> pairWithOffsetFn(@Name("coll") List<Object> values, @N
         return result;
     }
 
-    @Procedure
-    @Description("apoc.coll.pairWithOffset(values, offset) - returns a list of pairs defined by the offset")
+    @Procedure("apoc.coll.pairWithOffset")
+    @Description("Returns a list of pairs defined by the offset.")
     public Stream<ListResult> pairWithOffset(@Name("values") List<Object> values, @Name("offset") long offset) {
```
Comment (Lojjs) [inline core/src/main/java/apoc/coll/Coll.java:969]: I noticed that the first parameter is named `coll` in the 3 different places it is listed in docs, but `values` here, and in the signature and input parameters on the docs page. Same applies to `apoc.coll.partition` above.

Not sure which way we would like to change it, but should be consistent.

File: core/src/main/java/apoc/nodes/Nodes.java (hunk 1 above)
Comment (Lojjs) [reply core/src/main/java/apoc/nodes/Nodes.java:656]: So this one was not documented at all?

**Review by Lojjs — COMMENTED** [review]
Just one comment which might be more of a docs thing

File: core/src/main/java/apoc/load/LoadArrow.java [hunk 3]
```diff
@@ -109,7 +109,7 @@ public Stream<MapResult> stream(
     }
 
     @Procedure(name = "apoc.load.arrow")
-    @Description("apoc.load.arrow(fileName, config) - imports nodes and relationships from the provided file")
+    @Description("Imports nodes and relationships from the provided arrow file.")
```
Comment (Lojjs) [inline core/src/main/java/apoc/load/LoadArrow.java:112]: The docs PR did not have the word 'arrow' there if you want to be consistent, but on the other hand I think it is good to include it...

Comment (gem-neo4j) [reply core/src/main/java/apoc/load/LoadArrow.java:112]: Ah hehe I'll make a PR to docs for this as I think it is good :)

**Review by Lojjs — APPROVED** [review]

**Pull request merged**